awx/ui_next/src/screens/Inventory/Inventory.jsx

```diff
@@ -36,7 +49,7 @@ function Inventory({ history, i18n, location, match, setBreadcrumb }) {
     }
 
     fetchData();
-  }, [match.params.id, setBreadcrumb]);
+  }, [match.params.id, location.pathname, setBreadcrumb]);
```

Comment:
I think our odds of finding similar bugs for other detail pages are pretty high. We should probably run through them and see if this sort of change need to be made elsewhere, too